examples/apps/aml/web-client/deployment/nginx.conf

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Banco Santander S.A.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+user       nginx;
+worker_processes  5;  ## Default: 1
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+worker_rlimit_nofile 8192;
+
+events {
+  worker_connections  4096;  ## Default: 1024
+}
+
+http {
+  include    /etc/nginx/mime.types;
+  index    index.html index.htm index.php;
+
+  default_type application/octet-stream;
+  log_format   main '$remote_addr - $remote_user [$time_local]  $status '
+    '"$request" $body_bytes_sent "$http_referer" '
```

Comment:
s/http_referer/http_referrer/

Reply:
https://stackoverflow.com/questions/36240145/how-to-use-serverhttp-referer-correctly-in-php

I think it is necessary to keep "http_referer", it is required by nginx.

Reply:
Yes you can keep the parameter. But you should spell it correctly. Spellit `$http_referrer` with two 'r` characters.